src/components/modals/WelcomeModal.tsx

```diff
@@ -35,13 +35,13 @@ const WelcomeModal = ({ setShowModal }: ISignInModal) => {
 	};
 
 	return (
-		<Modal isOpen={true} style={customStyles}>
-			<CloseButton onClick={closeModal}>
-				<Image src={closeIcon} alt='close' />
-			</CloseButton>
+		<Modal showModal setShowModal={setShowModal} fullScreen hideHeader>
 			<ModalGrid>
 				<BGContainer />
 				<ContentContainer>
+					<CloseButton onClick={closeModal}>
+						<Image src={closeIcon} alt='close' />
+					</CloseButton>
```

Comment:
It's better to move the close button to the Modal component.

Reply:
@MohammadPCh Thanks for the review. I made the changes you mentioned.